### PR TITLE
Add method ToEnumOrDefault

### PR DESCRIPTION
**Pull request by Kurnakova**
Try convert string value to enum or return default value if not possible. With option to ignore case.

### CODE REVIEW

**Review by kurnakovv — CHANGES_REQUESTED** [review]
@Kurnakova please write unit tests and docs

File: DocsReadMe.md [hunk 1]
```diff
@@ -313,3 +313,47 @@ IEnumerable<MyEnum> output = inputEnums.ToEnums<MyEnum>(); // or false if case i
...
+
+MyEnum enumFirst = stringValueFirst.ToEnumOrDefault<MyEnum>();
+MyEnum enumSecond = stringValueSecond.ToEnumOrDefault<MyEnum>(false);
+MyEnum enumThird = stringValueInvalid.ToEnumOrDefault<MyEnum>(MyEnum.Third);
+
+// output
+// enumFirst - MyEnum.First
+// enumSecond - MyEnum.Second
+// enumThird - MyEnum.Third
+```
```
Comment (kurnakovv) [inline DocsReadMe.md:358]: btw, mb you just copy-paste docs from issue?

File: tests/EnumConverter.UnitTests/EnumConverterTest.cs [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ﻿using EnumConverterLibrary.UnitTests.TestEnums;
+using Microsoft.VisualStudio.TestPlatform.Common.Utilities;
```
Comment (kurnakovv) [inline tests/EnumConverter.UnitTests/EnumConverterTest.cs:2]: useless using

File: tests/EnumConverter.UnitTests/EnumConverterTest.cs [hunk 3]
```diff
@@ -496,5 +497,66 @@ public void ToEnums_CanConvertStringsToEnumsWithIgnoreCaseEqualsFalse_MyEnums()
...
         }
+
+        [Fact]
+        public void ToEnumOrDefault_CanConvertStringValueToEnum_MyEnum()
+        {
+            // Arrange.
+            string stringValueFirst = "First";
+            string stringValueSecond = "Second";
+            string stringValueThird = "Third";
+            // Act.
```
Comment (kurnakovv) [inline tests/EnumConverter.UnitTests/EnumConverterTest.cs:508]: add empty line between comments and code like that
```
// comment
code

// comment
code
```

File: tests/EnumConverter.UnitTests/EnumConverterTest.cs [hunk 4]
```diff
@@ -496,5 +497,66 @@ public void ToEnums_CanConvertStringsToEnumsWithIgnoreCaseEqualsFalse_MyEnums()
...
+            Assert.IsType<MyEnum>(myEnumSecond);
+            Assert.IsType<MyEnum>(myEnumThird);
+
+            Assert.Equal(MyEnum.First, myEnumFirst);
+            Assert.Equal(MyEnum.Second, myEnumSecond);
+            Assert.Equal(MyEnum.Third, myEnumThird);
+        }
+
+        [Fact]
+        public void ToEnumOrDefault_CanConvertStringToEnumsWithIgnoreCaseEqualsFalse_MyEnum()
```
Comment (kurnakovv) [inline tests/EnumConverter.UnitTests/EnumConverterTest.cs:523]: `ToEnums` -> `ToEnum` in name

**Review by kurnakovv — CHANGES_REQUESTED** [review]

File: DocsReadMe.md (hunk 1 above)
Comment (Kurnakova) [reply DocsReadMe.md:358]: I did copy-paste the text, but I added the formating signs myself. Should I have done it differently?

Comment (kurnakovv) [reply DocsReadMe.md]: Yes, you could copy the text from issue description (origin without markdown), but OK

**Review by kurnakovv — APPROVED** [review]

**Comment by kurnakovv** [comment]
@Kurnakova LGTM, thank you :)

**Pull request merged**